tests/e2e/simple_consensus.go

```diff
@@ -104,9 +118,22 @@ func SimpleConsensusTest(sc *RuntimeScenario, log *logging.Logger, conn *grpc.Cl
 		return fmt.Errorf("ensuring bob deposit consensus event: %w", err)
 	}
 
+	log.Info("wait for changes to propagate to client node")
```

Comment:
After the consensus layer event is emitted the runtime also needs to process the deposit. Let's not use `time.Sleep`, wait for the next round (or specific accounts transfer/mint/burn runtime event) instead.